<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMenuPList</class>
 <widget class="QWidget" name="FormMenuPList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormMenuPList</string>
  </property>
  <widget class="QPushButton" name="btn_sel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_copy">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>复制到...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>560</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="linedown">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>420</y>
     <width>124</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/down.png</normaloff>:/rc/icon/down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lineup">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>124</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/up.png</normaloff>:/rc/icon/up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pageup">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>20</y>
     <width>124</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/pageup.png</normaloff>:/rc/icon/pageup.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pagedown">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>490</y>
     <width>124</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/pagedown.png</normaloff>:/rc/icon/pagedown.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>850</width>
     <height>530</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>118</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>程序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>产品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>目标值</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>上限值</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>下限值</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>速度</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
